ui/component/claimPreview/view.jsx

```diff
@@ -347,7 +347,7 @@ const ClaimPreview = forwardRef<any, {}>((props: Props, ref: any) => {
             <>
               {!pending ? (
                 <NavLink aria-hidden tabIndex={-1} {...navLinkProps}>
-                  <FileThumbnail thumbnail={thumbnailUrl}>
+                  <FileThumbnail uri={uri} thumbnail={thumbnailUrl}>
```

Comment:
@jessopb I'm a bit concern on all the files I had to include the `uri` prop. since based on a few condition, it seems like several uris are being used. For example, in this very same file we got something like:

```js
                  <FileThumbnail uri={uri} thumbnail={thumbnailUrl}>
                    <div className="claim-preview__hover-actions">
                      {isPlayable && <FileWatchLaterLink focusable={false} uri={repostedContentUri} />}
                    </div>
                    {/* @if TARGET='app' */}
                    <div className="claim-preview__hover-actions">
                      {claim && !isCollection && (
                        <FileDownloadLink focusable={false} uri={canonicalUrl} hideOpenButton hideDownloadStatus />
                      )}
                    </div>
                    {/* @endif */}
                    <div className="claim-preview__file-property-overlay">
                      <PreviewOverlayProperties uri={uri} />
                    </div>
                  </FileThumbnail>
```

Notice all those conditionals to check which `uri` to use. Is this something we may cause issues?